assets/css/base/_variables.scss

```diff
@@ -117,16 +51,11 @@ $theme-logo-color-secondary:        #FF7701;
 $theme-logo-font-size:              40px;
 $theme-logo-font-weight:            800;
```

Comment:
Is this still in use? If not cleanout please ;)

Reply:
login page still